simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/Utils.java

```diff
@@ -0,0 +1,21 @@
+package io.prometheus.client.dropwizard;
+
+import java.util.regex.Pattern;
+
+public class Utils {
+    private static final Pattern METRIC_NAME_RE = Pattern.compile("[^a-zA-Z0-9:_]");
+
+    /**
+     * Replace all unsupported chars with '_', prepend '_' if name starts with digit.
+     *
+     * @param dropwizardName original metric name.
+     * @return the sanitized metric name.
+     */
+    public static String sanitizeMetricName(String dropwizardName) {
```

Comment:
This is already available from Collector

Reply:
Yup..removed thanks